R/createYaml.R

```diff
@@ -0,0 +1,161 @@
+#' Creates a yaml file storing the parameters that are used for creating the PTXQC report, 
+#' optionally returns these parameters and a list of qc-Metrics
+#'
+#' @param yc A yaml class object created by YAMLClass$new()
+#' @param path path to an (empty) yaml file
+#' @param param list of parameters sorted by names (important for shiny application)

```

Comment:
leave out the shiny comment (could be important for a number of future uses).

`... if NULL, will be populated with defaults`.

Ideally, name the parameters which are valid in the function description above (otherwise, one needs to inspect source code to know them).

Reply:
A minor problem here is when the given param is not NULL, but incomplete.

Solution: populate a `default_param` (as done in the NULL case currently) and update that default with matching values from `param`.